src/item.cpp

```diff
@@ -4457,7 +4480,13 @@ int item::get_encumber_when_containing(
 
     // Non-rigid items add additional encumbrance proportional to their volume
     if( !type->rigid ) {
-        encumber += contents_volume / 250_ml;
+        if( t->max_encumber != 0 ) {
+            // Cast up to 64 to prevent overflow. Dividing before would prevent this but lose data.
+            encumber += static_cast<int64_t>( t->max_encumber - t->encumber ) * contents_volume.value() /
+                        get_total_capacity().value();
```

Comment:
To silence that error, I suggest you add a check that this is not 0.
You can either use a ternary, like I do in this suggestion, or add a  check somewhere earlier.
1 is just my suggestion to keep it the same, not sure what you'd want.
```suggestion
                        get_total_capacity().value() > 0 ? get_total_capacity().value() : 1;
```
I'd suggest you don't use a ternary though, it's a bit long and hard to read.

Reply:
If you really think it's a thing that should never happen, you might want to have a full `if` statement so that you can put a `debugmsg` in the bad branch.

Reply:
Hypothetically it is possible, but the point of non-rigid objects is they change size with storage. I could just store get_total_capacity().value(), and check if before hand in the if statement.